php-practice/section1.php

```diff
@@ -179,6 +214,15 @@
 // (Hint: The syntax is sort of similar to PROBLEM 6)
 // =======================================================================
 
+$myAssocArray = [
+	'id' => 5, 
```

Comment:
Instead of re-creating the array,  show me how to add only the new element to the already created array.